Automattic/wp-calypso
# FSE: add colorized output to Jest unit tests output via lerna

Previously test output was not colorized. This made it much more difficult to decypher test output and distinguish from normal lerna output. 

Adding the [`--colors` flag to Jest](https://jestjs.io/docs/en/cli#--colors) fixes this.

> Forces test results output highlighting even if stdout is not a TTY.

#### Changes proposed in this Pull Request

Added colorized output when running FSE unit tests via `lerna`.

#### Testing instructions

Before checking-out  this PR branch:

* Run the FSE test suite - `npx lerna run test:js:watch --scope='@automattic/full-site-editing' --stream`
* See output is not colorized:

![Screen Shot 2020-03-02 at 10 06 25](https://user-images.githubusercontent.com/444434/75666740-4579f580-5c6e-11ea-9b7f-ae87eb9bd933.png)

After checking-out  this PR branch:

* Run the FSE test suite - `npx lerna run test:js:watch --scope='@automattic/full-site-editing' --stream`
* See output **_is_** now colorized:

![Screen Shot 2020-03-02 at 10 04 17](https://user-images.githubusercontent.com/444434/75666768-5296e480-5c6e-11ea-8f48-173fc08df08a.png)

🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉
